chihiro_todo/chihiro_jquery_todo/jquery.js

```diff
@@ -0,0 +1,190 @@
+let underEditedPersonId = undefined;
+
+//新規タスクをタスクリストに表示させる関数
+$(function () {
+    $('#task_submit').click(function() {
+        const taskElm =$('<li>').text($('#task_value').val()); 
+        taskElm.appendTo($('#todo_area'));
+        const selectedPersonElm = $('#person_select option:selected')
+        const taskPersonSpanElm =$('<span data-task-person-id= '+ selectedPersonElm.val() +' >').text(selectedPersonElm.text());
+        taskPersonSpanElm.appendTo(taskElm);
+        appendButtonByStatus(taskElm);
+        $('#task_value').val("");
+    })
+});
+
+//デフォルトの担当者をPersonListに表示させる関数
+$(function() {
+    $('#person_select option').each(function() {
+        appendNewPerson($(this).text(),$(this).val())
+    })
+})
+
+//新規担当者をPersonListに表示させる関数
+$(function() {
+    $('#person_submit').click(function() {
+        const createValue = new Date().getTime().toString()
```

Comment:
@chihiro-kakizaki 
`createValue`だと、`値を作る`っていうメソッド名みたいに見えてしまうので、変数名にするなら`createdValue`かな！
ただ、`作られた値`っていうのも抽象的すぎるから、素直に、`peresonId`で良いのではないかと思います！
```suggestion
        const personId = new Date().getTime().toString()
```

Reply:
@Shibata0806 
変数名がわかりづらいですね、、ご指摘ありがとうございます🙇‍♀️
以下のコミットにて修正しました。
[変数名createValueをpersonIdに変更](https://github.com/chihiro-kakizaki/todolist/pull/5/commits/c19140eb7aa173e147475dd1f58818a6281edf1e)